all_models/inflight_batcher_llm/tensorrt_llm/config.pbtxt

```diff
@@ -69,6 +69,13 @@ input [
     optional: true
     allow_ragged_batch: true
   },
+  {
+    name: "draft_acceptance_threshold"
```

Comment:
For this input, I checked https://github.com/NVIDIA/TensorRT-LLM/blob/main/cpp/include/tensorrt_llm/batch_manager/inferenceRequest.h#L37, I didn't find it, how do we use it?